src/main/scala/Configs.scala

```diff
@@ -153,6 +153,7 @@ class DefaultConfig extends Config (
       }
       case BuildRoCC => Nil
       case RoccNMemChannels => site(BuildRoCC).map(_.nMemChannels).foldLeft(0)(_ + _)
+      case RoccNPTWPorts => site(BuildRoCC).map(_.nPTWPorts).foldLeft(0)(_ + _)
```

Comment:
At some point in the past I think the parameters were changed to match some Scala style (camel case). I personally like the capital Cs but either way we should pick one